config/software/nodejs-binary.rb

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# nodejs-binary provides nodejs using the binary packages provided by
+# the upstream.  It does not install it into any of our standard
+# paths, so builds that require nodejs need to add embedded/nodejs/bin
+# to their path.
+#
+# Since nodejs is often a build-time-only dependency, it can then be
+# easily removed with remove-nodejs.
+#
+name "nodejs-binary"
+default_version "6.7.0"
+
+license "MIT"
+license_file "LICENSE"
+skip_transitive_dependency_licensing true
+
+version "6.7.0" do
+  source_hash = if ppc64le?
+                  "bc5fdfbe25a8fb0367ea7bf5b639ffb9bfd27ceb15f9d27579d6471ca585ff75"
+                elsif ppc64?
+                  "6ee874b6567f7f06708f6ccb66a27dc7317b4c493b7a6c3bb49c1e53c4bdf31e"
+                else
+                  "09263a844c31933c6f31e576e580faf01d3bbb056efb8713388dc8d09674f8c2"
+                end
+  source sha256: source_hash
+end
+
+arch_ext = if ppc64le?
+             "ppc64le"
+           elsif ppc64?
+             "ppc64"
+           else
+             "x64"
+           end
+
+source url: "https://nodejs.org/dist/v#{version}/node-v#{version}-linux-#{arch_ext}.tar.xz"
```

Comment:
le sigh:

```
[NetFetcher: nodejs-binary] I | 2016-10-03T09:24:10-04:00 | $ gtar Jxf /var/cache/omnibus/cache/node-v6.7.0-linux-x64.tar.xz -C/var/cache/omnibus/src/nodejs-binary
                          I | 2016-10-03T09:24:10-04:00 | gtar: invalid option -- J
                          I | 2016-10-03T09:24:10-04:00 | Try `gtar --help' or `gtar --usage' for more information.
The following shell command exited with status 64:

    $ gtar Jxf /var/cache/omnibus/cache/node-v6.7.0-linux-x64.tar.xz -C/var/cache/omnibus/src/nodejs-binary
```

Luckily it looks like upstreams provides tar.gz as well: https://nodejs.org/download/release/v6.7.0/